database/factories/BookingFactory.php

```diff
@@ -11,40 +11,27 @@
 
 class BookingFactory extends Factory
 {
-    /**
-     * The name of the factory's corresponding model.
-     *
-     * @var string
-     */
     protected $model = Booking::class;
 
-    /**
-     * Define the model's default state.
-     *
-     * @return array
-     */
     public function definition()
     {
         return [
             'slot_number'           => $this->faker->numberBetween(0, 10),
+            'user_id'               => randomOrCreate(app('user')),
+            'order_id'              => randomOrCreate(app('order')),
             'total_participants'    => $this->faker->numberBetween(0, 10),
             'total_amount'          => $this->faker->numberBetween(0, 10),
             'amount'                => $this->faker->numberBetween(0, 10),
             'total_taxes'           => $this->faker->numberBetween(0, 10),
             'total_fees'            => $this->faker->numberBetween(0, 10),
             'booking_category_id'   => randomOrCreate(app('booking_category')),
-            'rate_id'               => randomOrCreate(app('booking_category')), //TODO change class
+            'rate_id'               => randomOrCreate(app('rate')),
```

Comment:
if we create the relation with `rate` here, but in the `rateFactory` we have a relation with 
            'booking_id'            => randomOrCreate(app('booking')),

we are creating an infinite loop @sl0wik @drewroberts

Reply:
in this commit https://github.com/tipoff/bookings/pull/74/commits/1746148fb5a6b5559fda080854dd15b331a99992 I resolve the issue, let me know if this is ok, otherwise it can be reverted

Reply:
Thank you. Is this correct @sl0wik? Or did you have another intention for the Rate model?

Reply:
@wolfrednicolas booking got rate. Rate got bookings. Rate should not have single booking.